src/assertions/OrderAssertions.ts

```diff
@@ -182,6 +182,21 @@ export class OrderAssertions {
     );
   }
 
+  /* ============ Private Helpers =============== */
+
+  private calculateLiquidityFills(
```

Comment:
Also, could re-order the methods so that the highest-level private methods are on top (the methods that call other private methods)?

For example, `assertOrdersValidity` and `assertSufficientMakerTokensForOrders` should be on the top, as they are called by  public function `assertLiquidityValidity`.